packages/launcher/lib/linux/index.ts

```diff
@@ -73,6 +73,26 @@ export function getVersionString (path: string) {
   .then(tap(partial(log, ['stdout: "%s"'])))
 }
 
+export function getPathData (pathStr: string): { path: string, browserKey?: string } {
+  const test = new RegExp(/^.+:(.+)$/)
+  const res = test.exec(pathStr)
+  let browserKey = ''
+  let path = pathStr
+
+  if (res) {
+    const pathParts = path.split(':')
```

Comment:
Windows's NTFS and FAT32 filesystems prohibit using `:` inside of a filename, which is why this separator character was chosen:

* https://docs.microsoft.com/en-us/windows/win32/fileio/naming-a-file#naming-conventions
* https://en.wikipedia.org/wiki/Filename#Comparison_of_filename_limitations

However, in Linux ext4 filesystem, it is perfectly fine to create a filename with a colon in it:

```
➜ ~ echo "foo" > test:file
➜ ~ cat test:file
foo
```

I think this reinforces the idea that this feature should be Windows-only, so we can *avoid* having to create additional logic to make paths containing colons still work.